examples/howto/plot_hnn_animation.py

```diff
@@ -0,0 +1,67 @@
+"""
+================================
+06. Animating HNN simulations
+================================
+
+This example demonstrates how to animate HNN simulations
+"""
+
+# Author: Nick Tolley <nicholas_tolley@brown.edu>
+
+
+###############################################################################
+# First, we'll import the necessary modules for instantiating a network and
+# running a simulation that we would like to animate.
+import os.path as op
+import hnn_core
+from hnn_core import jones_2009_model, simulate_dipole, read_params
+from hnn_core.network_models import add_erp_drives_to_jones_model
+
+###############################################################################
+# We begin by instantiating the network. For this example, we will reduce the
+# number of cells in the network to speed up the simulations.
+hnn_core_root = op.dirname(hnn_core.__file__)
+params_fname = op.join(hnn_core_root, 'param', 'default.json')
+params = read_params(params_fname)
+params.update({'N_pyr_x': 3, 'N_pyr_y': 3})
+net = jones_2009_model(params)
+
+# Note that we move the cells further apart to allow better visualization of
+# the network (default inplane_distance=1.0 µm).
+net.set_cell_positions(inplane_distance=300)
+
+###############################################################################
+# The :class:`hnn_core.viz.NetworkPlotter` class can be used to visualize
+# the 3D structure of the network.
+from hnn_core.viz import NetworkPlotter
+
+net_plot = NetworkPlotter(net)
+net_plot.fig
```

Comment:
I don't see a figure in `ipython`. Maybe you need a `plt.show` or `plt.draw` ? Maybe `net_plot.show()` ?

Reply:
This shows up in circleCI docs: 
![image](https://github.com/jonescompneurolab/hnn-core/assets/55253912/dbdee874-c9f1-4012-9937-6aff40266de9)

honestly this issue of when we do and don't need to use `plt.show()` is really opaque to me, it is only a problem sometimes on some certain systems...